backend/tox.ini

```diff
@@ -0,0 +1,17 @@
+[tox]
+envlist = 
+    py3,
+    lint,
+skipsdist = True
+
+[testenv:py3]
+deps = -r requirements.txt
+commands = pytest -v
+
+[pytest]
+DJANGO_SETTINGS_MODULE = backend.settings
+python_files = test*.py
+
+[testenv:lint]
+whitelist_externals = black
+commands = black api/ backend/
```

Comment:
Currently deciding whether we should keep this or nah. It essentially runs `pytest` and `black` in one go while providing some functionalities from GitHub actions. I commented out the `tox` reference within the `make backend` command.